examples/api/test/widgets/scroll_view/custom_scroll_view.1_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
```

Comment:
```suggestion
```

```
info • The import of 'package:flutter/gestures.dart' is unnecessary because all of the
       used elements are also provided by the import of 'package:flutter/material.dart'
```